indigo-plugin/indigo-plugin/src/indigoplugin/generators/AssetListing.scala

```diff
@@ -245,4 +260,10 @@ object AssetListing {
       "tiff"
     )
 
+  val FontFileExtensions: Set[String] =
+    Set(
+      "ttf",
+      "woff",
+      "woff2"
```

Comment:
`eot` is another one. Arguably we can ignore it, but then, the image list includes `tiff` files so I think the pattern is to do the official thing and leave the user to sort it out.

Reply:
Yup, agree :)